.travis.yml

```diff
@@ -1,19 +1,19 @@
 # This file is part of Requirements-Builder
-# Copyright (C) 2015, 2016, 2017, 2018 CERN.
+# Copyright (C) 2015, 2016, 2017, 2018, 2019 CERN.
 #
 # Requirements-Builder is free software; you can redistribute it and/or
 # modify it under the terms of the Revised BSD License; see LICENSE
 # file for more details.
 
-sudo: false
+dist: xenial
+sudo: true
 
 language: python
 
 python:
   - "3.7-dev"
```

Comment:
Should test the production release of 3.7 and the dev release of 3.8 given that it is late beta.

Reply:
My intent was to let that part of #40

Reply:
You got it...  Keep going.